tools/accuracy_checker/accuracy_checker/metrics/README.md

```diff
@@ -81,8 +81,13 @@ More detailed information about calculation segmentation metrics you can find [h
   * `uninterpolated_auc` - should area under precision recall curve be computed using trapezoidal rule or directly.
 * `pairwise_accuracy` - pairwise accuracy for object reidentification. Supported representations: `ReIdentificationClassificationAnnotation`, `ReIdentificationPrediction`.
   * `min_score` - min score for determining that objects are different. You can provide value or use `train_median` value which will be calculated if annotations has training subset.
+  * `distance_mode` - allows to choose one of the distance calculation mode (optional).
+  * `subtract_mean` - allows to subtract mean calculated on train embeddings before calculating the distance(optional, default - `False`).
 * `pairwise_accuracy_subsets` - object reidentification pairwise accuracy with division dataset on test and train subsets for calculation mean score. Supported representations: `ReIdentificationClassificationAnnotation`, `ReIdentificationPrediction`.
   * `subset_number` - number of subsets for separating.
+  * `min_score` - min score for determining that objects are different. You can provide value or use `train_median` value which will be calculated if annotations has training subset.
```

Comment:
I do not see info about best_train_threshold

Reply:
Done